02-lsp_extension/02.01-lsp_exercises1.ipynb

```diff
@@ -0,0 +1,523 @@
+{
```

Comment:
<p>"The exact key or shortcut depends on your code editor or IDE. Typically, it is triggered by <strong>typing a trigger character, such as a period "." or an opening parenthesis "("</strong>" --&gt; can you tell us what the default is for JupyterLab specifically?</p><p> </p><p>I think it would be good to include a gif of what code completion/navigation loks like if possible. Making them can feel rote but they add a lot of value I think.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nicole-brewer/jupyterlab-ide/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>